### PR TITLE
RavenDB-12444 Add top threads endpoint

### PR DESCRIPTION
**Pull request by karmeli87**

### CODE REVIEW

File: src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs [hunk 1]
```diff
@@ -13,6 +14,32 @@ namespace Raven.Server.Documents.Handlers.Debugging
 {
     public class ThreadsHandler : RequestHandler
     {
+        [RavenAction("/admin/debug/threads/top", "GET", AuthorizationStatus.Operator, IsDebugInformationEndpoint = true)]
```
Comment (ayende) [inline src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: Isn't this just runaway?

Comment (ayende) [reply src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: There we are using `thread.TotalProcessorTime.TotalMilliseconds`

Comment (grisha-kotler) [reply src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: It doesn't provide the actual CPU usage. @karmeli87 maybe we can combine the 2 endpoints?

Comment (karmeli87) [reply src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: I think those are 2 different measures. `runaway` measuring from the start-up, while `top` is only for the last second.

But we can merge them if you like, the question is how we will order them?

Comment (arekpalinski) [reply src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: Reminder - @grisha-kotler ?

**Comment by ayende** [comment]
Reminder @grisha-kotler

File: src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs (hunk 1 above)
Comment (grisha-kotler) [reply src/Raven.Server/Documents/Handlers/Debugging/ThreadsHandler.cs]: Merged both endpoints

**Review by ppekrol — APPROVED** [review]

**Comment by ppekrol** [comment]
@grisha-kotler I'm merging this. Please create a separate PR with adjustments.

**Pull request merged**